quelpa.el

```diff
@@ -1854,14 +1869,36 @@ nil."
   (when (quelpa-setup-p) ;if init fails we do nothing
     (let* ((quelpa-upgrade-p (if current-prefix-arg t quelpa-upgrade-p)) ;shadow `quelpa-upgrade-p'
            (quelpa-stable-p quelpa-stable-p) ;shadow `quelpa-stable-p'
-           (cache-item (if (symbolp arg) (list arg) arg)))
+           (cache-item (if (symbolp arg) (list arg) arg))
+           version)
       (quelpa-parse-plist plist)
       (quelpa-parse-stable cache-item)
-      (quelpa-package-install arg)
+      (when (setq version (quelpa-package-install arg))
+        (setq cache-item `(,(car cache-item)
+                           ,@(plist-put (cdr cache-item) :version version))))
       (quelpa-update-cache cache-item)))
   (quelpa-shutdown)
   (run-hooks 'quelpa-after-hook))
 
+;;;###autoload
+(define-minor-mode quelpa-mode
+  "Global minor mode for Quelpa."
+  :init-value t
+  :group 'quelpa
+  :global t
+  :lighter " Quelpa"
```

Comment:
I think we should omit the lighter, because I don't think it's useful to show it in the mode line.  What do you think?

Reply:
yeah, it's more like a distraction I would say